virtcontainers/container.go

```diff
@@ -886,38 +886,9 @@ func (c *Container) create() (err error) {
 		}
 	}
 
-	var (
```

Comment:
I need to go back to the discussion on the  agent PR to see its progress, but adding a note here that the agent PR was blocked due to the inherent delay introduced by the kernel while hotplugging pcie devices.